README.md

```
@@ -210,6 +211,41 @@ Use `context.json()` to return structured JSON output with advanced control over
 
 For detailed information about available JSON fields and their behavior, see the [official documentation](https://docs.anthropic.com/en/docs/claude-code/hooks#advanced:-json-output).
 
+### Async JSON Output (Experimental)
+
+> [!WARNING]
+> This behavior is undocumented by Anthropic and may change.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)